scripts/rest_examples/reservation-create-gcp.http

```diff
@@ -10,6 +10,7 @@ X-Rh-Identity: {{identity}}
   "image_id": "https://www.googleapis.com/compute/v1/projects/rhel-cloud/global/images/rhel-9-v20230411",
   "amount": 1,
   "machine_type": "e2-micro",
+  "launch_template_name": "instance-template-1",
```

Comment:
Maybe create this a variable so devs can change template, also the default could be empty - API should ignore an empty template name and treat it as "nothing was provided". Just an idea.